### PR TITLE
{tools}[dummy/dummy] itac v2019.4.036

### PR DESCRIPTION
**Pull request by ekieffer**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ekieffer** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-003 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/331742b76b83ab197918c08b9bcd77e6 for a full test report.

File: easybuild/easyconfigs/i/itac/itac-2019.4.036.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description = """The Intel Trace Collector is a low-overhead tracing library that performs
+ event-based tracing in applications. The Intel Trace Analyzer provides a convenient way to monitor application
+ activities gathered by the Intel Trace Collector through graphical displays. """
+
+toolchain = SYSTEM
+
+sources = ['l_itac_p_%(version)s.tgz']
+checksums = ['8994f20c12ea314760482314677d278628cbe8bd79625b302b4d02c8ae69ca53']
+
+dontcreateinstalldir = 'True'
```
Comment (smoors) [inline easybuild/easyconfigs/i/itac/itac-2019.4.036.eb:14]: @ekieffer `dontcreateinstalldir` should be boolean:
```suggestion
dontcreateinstalldir = True
```

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node378.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/0bd789adbdd9fd3046beb33a1bf1c6a1 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node365.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/2e081b86574d464d24be133f99f5db89 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node364.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/f0076f90fb7dff65213ca6fd327fb094 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node364.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/ba302f14a6fcdaddf8f4b59753496cdf for a full test report.

**Comment by smoors** [comment]
Going in, thanks @ekieffer!

**Pull request merged**